<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeCreateAutoPCBHoleAlignerFormClass</class>
 <widget class="QDialog" name="ChangeCreateAutoPCBHoleAlignerFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeCreateAutoPCBHoleAlignerForm</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>441</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ライブラリ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frameLibFolder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetLibList">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>471</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>名称</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditLibID">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>12345</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditLibName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>311</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Library Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>471</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>ライブラリプロパティ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonOK">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="icon">
      <iconset resource="AutoPCBHoleAligner.qrc">
       <normaloff>:/Qtres/flash_swf_64.PNG</normaloff>:/Qtres/flash_swf_64.PNG</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="ButtonCancel">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>220</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="AutoPCBHoleAligner.qrc">
       <normaloff>:/Qtres/flash_64.PNG</normaloff>:/Qtres/flash_64.PNG</iconset>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>451</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QCheckBox" name="checkBoxCheckOutside">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>明るい領域に囲まれているかの判定</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxStartupExecute">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>131</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>デフォルトで実行</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditThresholdDiff">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>最大直径ドット数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>最小直径ドット数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>周囲探索ドット数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditMaxSearch">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>周縁ドット数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditFringeDot">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>穴領域膨張量</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string>輝度変化閾値</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditMaxDiameter">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditExpandForPickup">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="EditMinDiameter">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AutoPCBHoleAligner.qrc"/>
 </resources>
 <connections/>
</ui>
